tracing/src/main/java/com/palantir/tracing/CloseableTracer.java

```diff
@@ -32,14 +33,14 @@ private CloseableTracer() {}
     /**
      * Opens a new {@link SpanType#LOCAL LOCAL} span for this thread's call trace, labeled with the provided operation.
      */
-    public static CloseableTracer startSpan(String operation) {
+    public static CloseableTracer startSpan(@Safe String operation) {
```

Comment:
Can you explain what does this actually do?

Reply:
It's only there for documentation, has no impact at present. Eventually I'd like to build static analysis which validates the provided data is not unsafe, this lays some foundation.